src/main/java/decodex/recipes/Recipe.java

```diff
@@ -0,0 +1,68 @@
+package decodex.recipes;
+
+import java.util.ArrayList;
+import java.util.logging.Logger;
+
+import decodex.Decodex;
+import decodex.data.exception.RecipeException;
+import decodex.modules.Module;
+
+/**
+ * The Recipe class manages a list of Modules to be run sequentially.
+ */
+public class Recipe {
+
+    private Logger logger = Decodex.logger;
+
+    private String name;
+    private ArrayList<Module> moduleList = new ArrayList<>();
+
+    public Recipe(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
```

Comment:
Perhaps check if the recipe name is blank and throw an exception? Also if you are adding the check, make sure to create a test case for this too